Library/Formula/subliminal.rb

```diff
@@ -15,93 +13,106 @@ class Subliminal < Formula
 
   depends_on :python if MacOS.version <= :snow_leopard
 
-  resource "charade" do
-    url "https://pypi.python.org/packages/source/c/charade/charade-1.0.3.tar.gz"
-    sha1 "6bf65ae4b60d13e89bd7afe1704eef5bd0b787bc"
-  end
-
-  # Don't upgrade >0.7.0 - Package incompatible with above.
-  resource "guessit" do
```

Comment:
You bumped ` guessit `. Is upstream Subliminal compatible with newer versions of ` guessit ` finally or?